packages/code-du-travail-frontend/src/piwik.js

```diff
@@ -2,20 +2,26 @@ import ReactPiwik from "react-piwik";
 import { Router } from "../routes";
 import getConfig from "next/config";
 
-const onRouteChangeComplete = () => {
-  ReactPiwik.push(["trackPageView"]);
+let piwik;
```

Comment:
I don't like this piwik here. I wouldn't need it if the callback is directly declared where it's used ?